docs/pages/bssw/bssw_wikize_refs.md

```diff
@@ -43,9 +43,9 @@ at the end of the markdown file, of the form
 - `LAB`: (**REQUIRED**) Alphanumeric *label* for the reference
   - **Note**: The post-processing script renumbers them 1...N
 - `URL`: (**REQUIRED**) The URL for the reference
-- `TIT`: (*optional*) A title for the reference (appears as tool-tip during hover)
-- `{BIB}`: (*optional*) Full bibliographic information for the reference
-  - Here, we are using the `{BIB}` field, embedded in the [link's title](https://www.markdownguide.org/basic-syntax#adding-titles) text as a notational *extension*.
+- `TIT`: (*optional*) A title for the reference (will appear as tool-tip during hover)
```

Comment:
I suggest "will also appear" to reinforce the point that TIT is used in the reference as well.